gh-131831: Implement PEP 758 – Allow except and except* expressions without parentheses (#131833)
python/cpython: Doc/reference/compound_stmts.rst

and information on using the :keyword:`raise` statement to generate exceptions
may be found in section :ref:`raise`.

> .. versionchanged:: 3.13

Reviewer: ```suggestion
.. versionchanged:: next
```

it matches any exception.

For an :keyword:`!except` clause with an expression, the
expression must evaluate to an exception type or a tuple of exception types. Parentheses

> can be dropped if multiple exception types are provided and the ``as`` clause is not used.

Reviewer: Since parentheses around tuples are optional elsewhere in the language, maybe it would be clearer to phrase this as emphasizing the case where they are required? Something like "If multiple exception types are provided as a tuple expression, then parentheses are required if the ``as`` clause is used."?

Author: Done

Reviewer: I think this got overwritten by another push (ignore if that was intentional)

> expression must evaluate to an exception type or a tuple of exception types. Parentheses
> can be dropped if multiple exception types are provided and the ``as`` clause is not used.

Author: ```suggestion
expression must evaluate to an exception type or a tuple of exception types. If multiple exception types are provided as a tuple expression, then parentheses are required if the ``as`` clause is used.
```
